chart/values.yaml

```diff
@@ -132,6 +132,7 @@ php:
 
 # Configuration for everything that runs in shell container.
 shell:
+  enabled: false
```

Comment:
Most of the settings are optimised for development environments, and in that context a shell container makes a lot of sense, so I would have it enabled by default.

One of the reasons for not having a shell container enabled by default would be to speed up the deployment. In that case we should also prevent the container from being built in the first place, but that's definitely a different task.

Reply:
reenabled